ragna/_ui/logout_page.py

```diff
@@ -0,0 +1,22 @@
+import panel as pn
+import param
+
+
+class LogoutPage(pn.viewable.Viewer, param.Parameterized):
+    def __init__(self, api_wrapper, **params):
+        super().__init__(**params)
+        self.api_wrapper = api_wrapper
+
+        self.api_wrapper.auth_token = None
+
+    def __panel__(self):
+        # Usually, we do a redirect this way :
+        # >>> pn.state.location.param.update(reload=True, pathname="/")
+        # But it only works once the page is fully loaded.
+        # So we render a javascript redirect instead.
+        return pn.pane.HTML(
+            """<script>
+                            document.cookie = "auth_token=; expires=Thu, 01 Jan 1970 00:00:00 UTC; path=/";
```

Comment:
What is the expiration in the past here?

Reply:
I was hoping the context would be explicit enough but I've pushed a comment to clarify : to remove a cookie, you have to set its expiry date to the past to force it to expire.